modelskill/metrics.py

```diff
@@ -548,8 +549,8 @@ def peak_ratio(
 ) -> float:
```

Comment:
All parameters should be documented in the docstring.

Reply:
But they are

https://github.com/DHI/modelskill/blob/12d234acfde1576377fce3e2a7eec222e9d47041/modelskill/metrics.py#L543-L567

Unless I am missing something?

Reply:
Is AAP a int or a float?
Seems like we are missing `obs`, `model` in all docstrings for all metrics 😳.

Reply:
I thought it was INT, but actually, it can be float, I will change it.
And yes, we are missing `obs` and `model` in all metrics, but maybe that is given at this point